addons/udes_stock/models/stock_picking_type.py

```diff
@@ -443,6 +457,8 @@ def _prepare_info(self):
             - u_reserve_pallet_per_picking: boolean
             - u_max_reservable_pallets: int
             - u_selection_method: string
+            - u_check_package_type: boolean
+            - u_default_package_type_id: id
```

Comment:
nit: should this say `int` like `default_location_dest_id`? I prefer `id` as it's more informative but for consistency `int` wins

Reply:
Yep, that wasn't intentional! Good spot. :)

Reply:
Fixed. :)